ci-operator/config/openshift/cluster-capi-operator/openshift-cluster-capi-operator-main.yaml

```diff
@@ -7,6 +7,10 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-9-golang-1.22-openshift-4.17
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.16"
```

Comment:
What does this tag imply?
Will this be automatically bumped?

Reply:
I will need to check this , at present we are manually doing it . This is the release version of our test repo.
 @liangxia @kuiwang02 let me know if I am not correct.

Reply:
Yes, we are changing this manually. 
This indicate which image/branch the ginkgo e2e tests (openshift-extended-test) is using.

Reply:
Ok got it. Do you have reminders set up to remind you to do this then?

Reply:
yes when the branch cuts ( code freeze for openshift ) , we also branch out from our master and then update . So for example , our master at present will become 4.17 , when openshift master cuts to 4.17 , not exactly the same time but close to it .